app/models/aggregation.rb

```diff
@@ -14,8 +14,11 @@ class Aggregation < ActiveRecord::Base
   validate :aggregation, :workflow_version_present
 
   def self.scope_for(action, user, opts={})
-    return super unless (action == :show || action == :index) && !user.is_admin?
-    joins(:workflow).merge(Workflow.scope_for(:update, user, opts))
+    if (action == :show || action == :index) && !user.is_admin?
+      joins(:workflow).merge(Workflow.scope_for(:update, user, opts))
```

Comment:
You could just remove https://github.com/camallen/Panoptes/blob/public_aggregations_rebase/app/controllers/api/v1/aggregations_controller.rb#L3. People will get 404s on #show routes, and empty sets if there are no public aggregations for them to se.